sdk/tests/wallet/wallet_storage.rs

```diff
@@ -17,6 +20,16 @@ async fn check_existing_db() -> Result<()> {
 
     let wallet = Wallet::builder().with_storage_path(storage_path).finish().await?;
 
+    // Migrate old snapshots.
+    let _ = StrongholdAdapter::migrate_snapshot_v2_to_v3(
```

Comment:
SHould we not unwrap especially in a test?

Reply:
No, because we want to migrate if it was v2, but if it was v3 we don't want it to fail. It will fail later if there was an issue with migration anyway.